test/unit/validators/non_localhost_validator_test.rb

```diff
@@ -1,10 +1,12 @@
+# frozen_string_literal: true
+
 require 'test_helper'
 
-class ThreeScale::LocalhostValidatorTest < ActiveSupport::TestCase
+class NonLocalhostValidatorTest < ActiveSupport::TestCase
```

Comment:
Could be worth it adding an edge case to this test, something containing localhost? e.g. `http://mylocalhost/` Could give a false negative, depending on the regex.

Reply:
I didn't actually wrote that, just renamed it. But I guess you're right. It won't kill me enhancing it a bit since I'm here 😜

Reply:
I know, I know 😄 .. that's what happens when touching others' stuff hehe

Reply:
Done.